administrator/language/en-GB/plg_schemaorg_event.ini

```
@@ -0,0 +1,28 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_EVENT="Schema.org - Event"
+PLG_SCHEMAORG_EVENT_DESCRIPTION_LABEL="A Schema.org Type <a href=\"https://schema.org/Event\" target=\"_blank\">More information</a>"

```

```suggestion
PLG_SCHEMAORG_EVENT_DESCRIPTION_LABEL="A Schema.org Type <a href=\"https://schema.org/Event\" target=\"_blank\"  rel=\"noopener noreferrer\">More information</a>"
```

This is for security purposes. Sorry your mentors didnt inform you of this